modules/speller/default/suggest.cpp

```diff
@@ -74,7 +77,10 @@ using namespace std;
 
 namespace {
 
-  typedef vector<String> NearMissesFinal;
+  // The internal, C++ version of the external Suggest/AspellSuggest
+  // struct.  I don't use that struct because I want the convenience
+  // of automatically allocated space for the strings.
+  typedef vector<pair<String,int> > NearMissesFinal;
```

Comment:
Added. I thought that the comment meant to reuse the existing AspellSuggestion struct and was more involved.